README.md

```diff
@@ -40,23 +40,52 @@ listed per suite after the suite has finished.
 
 ### Filtering the results
 
+#### Include / exclude
+
 There are projects where it's important to avoid some step definitions to be
 detected. For instance, when a project wants to avoid scanning unused step
 definitions from the third-party packages/libraries and show only results from
-the custom code. The extension allows to configure a _regular expression_ filter
-in the `behat.yml` configuration file. The filter will only allow definitions
-whose context class name satisfies the regular expression:
+the custom code. The extension allows to configure a list of
+_regular expressions_ to include or exclude step definitions in the `behat.yml`
+configuration file. Expressions are compared against the FQCN + method name
+(`My\Namespace\ClassName::methodName`):
 
 ```yaml
 default:
   extensions:
     NicWortel\BehatUnusedStepDefinitionsExtension\Extension:
-      filter: '#\\MyProject\\Behat\\Contexts#'
+      filters:
+        include:
+          - 'MyProject\\Behat\\Contexts'
+          - 'OtherProject\\Behat\\(Foo|Bar)Context'
+        exclude:
+          - 'MyProject\\Behat\\Contexts\\FeatureContext'
+          - '::excludedMethod'
+          - 'OtherProject\\Behat\\FooContext::.+Method'
```

Comment:
Very nice improvement. But I think we should offer backwards compatibility for the legacy `filter` argument. Apart for making it work, I think we need to emit some visible warning error that `filter` is deprecated

Reply:
See https://github.com/nicwortel/behat-unused-step-definitions-extension/pull/7/commits/c3693164cf16412539f954f33fa35e8baf6d99a4
I kept the current logic to keep the regex delimiters in config, iterate through each item and preg_match.